functions/strategies/ProxyStrategy.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const proxyStrategy = async (req, store) => {
+  const proxyRes = await axios.post(
```

Comment:
```suggestion
  const { data } = await axios.post(
```

Reply:
I'm a big fan of the object destructuring approach. It's quite subjective :sweat_smile:

Reply:
Updated in the latest commit